examples/navigation/index.js

```diff
@@ -0,0 +1,30 @@
+import 'rxjs';
```

Comment:
I see. That makes sense. I'd add a comment though, because it reads funny to those of us who are used to only importing exactly what we need from rxjs.